packages/shared/src/hooks/useRegistration.ts

```diff
@@ -77,6 +80,8 @@ const useRegistration = ({
     queryKey: key,
     queryFn: () => initializeKratosFlow(AuthFlow.Registration),
     refetchOnWindowFocus: false,
+    refetchOnMount: false,
```

Comment:
We have disableRefetch helper that you can spread, not sure if this is leftover from local testing though?

Reply:
not leftover, it's a fix. otherwise the native auth breaks